qml/pages/VenueList.qml

```diff
@@ -74,6 +74,11 @@ BVApp.Page {
             currentIndex = -1
         }
 
+        onContentYChanged: {
```

Comment:
Aha, strange that we have to add this here, as well, now…
Good find! We don't want to have this guy (#113), not even in the first release…